meshtastic/tests/test_node.py

```
@@ -1360,6 +1364,14 @@ def test_getURL_requests_lora_when_local_config_empty(
     anode.channels = [primary, secondary, disabled]
     anode.requestConfig = MagicMock()  # type: ignore[method-assign]
 
+    def _populate_lora_and_return_true(attribute: str = "channels") -> bool:
```

Ruff (0.15.6)

[warning] Unused function argument: `attribute`

(ARG001)